Cannot be used with Gradle 6 & JRE 8

I am mainly opening this to show that the maintainers of this project are aware of the problem.

The underlying problem is https://github.com/robstoll/kbox/issues/26, which I am working on.

## How to reproduce the problem  
Use atrium 0.9 on a JRE version 8 with gradle 6.

## Describe the bug
The build will fail with an error message like this:

```
> Task :compileTestKotlin FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestKotlin'.
> Could not resolve all files for configuration ':testCompileClasspath'.
   > Could not resolve ch.tutteli.kbox:kbox:0.14.1.
     Required by:
         project : > ch.tutteli.atrium:atrium-cc-en_GB-robstoll:0.9.0 > ch.tutteli.atrium:atrium-core-api:0.9.0
      > Unable to find a matching variant of ch.tutteli.kbox:kbox:0.14.1:
          - Variant 'apiElements' capability ch.tutteli.kbox:kbox:0.14.1:
              - Incompatible attribute:
                  - Required org.gradle.jvm.version '8' and found incompatible value '9'.

              - Other attributes:
                  - Found org.gradle.category 'library' but wasn't required.
```

See, for example, https://github.com/jGleitz/string-notation/pull/14/checks?check_run_id=422070218

## Expected behaviour
The build succeeds.

--- a/build.gradle
+++ b/build.gradle
@@ -7,9 +7,9 @@ buildscript {
     }
     ext {
         // main
-        kbox_version = '0.14.2'
+        kbox_version = '0.14.3'
         kbox = { "ch.tutteli.kbox:kbox:$kbox_version" }
-        niok_version = '1.3.3'
+        niok_version = '1.3.4'
         niok = { "ch.tutteli.niok:niok:$niok_version" }
         kotlin_version = '1.3.61'
 
@@ -28,7 +28,7 @@ buildscript {
         mockito = { "com.nhaarman.mockitokotlin2:mockito-kotlin:$mockito_kotlin_version" }
 
         // project setup
-        tutteli_plugins_version = '0.32.1'
+        tutteli_plugins_version = '0.32.2'
         node_plugin_version = '1.3.1'
 
         // gh-pages.gradle
@@ -227,8 +227,8 @@ configure(subprojects - commonProjects - jsProjects - sampleProjects) {
         }
     }
 
-    sourceCompatibility=8
-    targetCompatibility=8
+    sourceCompatibility = 8
+    targetCompatibility = 8
 }
 
 //allow kotlin 1.3 for kotlin_1_3 modules

--- a/settings.gradle
+++ b/settings.gradle
@@ -1,7 +1,7 @@
 rootProject.name = 'atrium'
 
 buildscript {
-    gradle.ext.tutteli_plugins_version = '0.31.2'
+    gradle.ext.tutteli_plugins_version = '0.32.2'
     repositories {
         maven { url "https://plugins.gradle.org/m2/" }
     }
@@ -35,7 +35,7 @@ def bundleWithExtensionsAndSmokeTest = { self, String apiName ->
     if (JavaVersion.current() >= JavaVersion.VERSION_1_9) {
         self.folder(apiName) {
             _ "$apiName-smoke-test"
-            folder("extensions"){
+            folder("extensions") {
                 _ "$apiName-smoke-test-jdk8"
                 _ "$apiName-smoke-test-kotlin_1_3"
             }

